src/enrichment.cc

```diff
@@ -184,44 +175,44 @@ std::set<cyclus::BidPortfolio<cyclus::Material>::Ptr> Enrichment::GetMatlBids(
 
   if ((out_requests.count(tails_commod) > 0) && (tails.quantity() > 0)) {
     BidPortfolio<Material>::Ptr tails_port(new BidPortfolio<Material>());
-    
+
     std::vector<Request<Material>*>& tails_requests =
-      out_requests[tails_commod];
+        out_requests[tails_commod];
```

Comment:
Apparently when breaking a line, we put 4 spaces indent  (I found an example in growth_region.cc, line 60 as google C++ style was not explicit about this specific point)